id_ID/MimeTypeDisplay_id.properties

```diff
@@ -0,0 +1,54 @@
+# MimeTypeDisplay File properties
+# Nama yang mudah digunakan pengguna untuk menampilkan jenis pantomim.
+# Dokumentasi, Data, File arsip:
+application / pdf = Adobe PDF
```

Comment:
Should these be (for example)

`application/pdf`

instead of 

`application / pdf`

?